docs/main/user-guides/migration/legacy-storage-to-new-storage/cstor-to-replicated.md

```diff
@@ -0,0 +1,127 @@
+---
+id: cstor-to-replicated
+title: Migration from OpenEBS cStor to OpenEBS Replicated
+keywords:
+ - Migration from OpenEBS cStor to OpenEBS Replicated
+ - cStor to Replicated
+ - cStor to Mayastor
+ - Jiva to Replicated
+ - Jiva to Mayastor
+description: This section outlines the process of migrating OpenEBS cStor to OpenEBS Replicated.
+---
+
+:::info
+The following steps are an example about migrating from legacy storage to new storage.
+You can also migrate OpenEBS Jiva to OpenEBS Replicated using the steps below.
+:::
+
+## Assumptions
+
+- cStor is already deployed.
+- MongoDB Standalone is deployed as below using the cStor PVC. (Here, MongoDB Standalone is an example.)
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: cstor-pvc
+spec:
+  storageClassName: cstor-csi-disk
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+- For validation, some data has been inserted in the MongoDB as an example below:
+
+```
+db.admin.insertMany([{name: "Max"}, {name:"Alex"}])
+
+[
+  { _id: ObjectId('65eaafa01cd2b6de45285d86'), name: 'Max' },
+  { _id: ObjectId('65eaafa01cd2b6de45285d87'), name: 'Alex' }
+]
+```
+## Steps to migrate cStor to Replicated
+
+Follow the steps below to migrate OpenEBS cStor to OpenEBS Replicated (fka Mayastor).
```

Comment:
aka instead of fka ?

Reply:
The term "Mayastor" was formerly used, and since it has been renamed to "Replicated Engine", we have used **fka**. 

We usually use **aka** when the phrase or name is currently in use.